src/panels/config/zone/dialog-zone-detail.ts

```diff
@@ -236,13 +236,17 @@ class DialogZoneDetail extends LitElement {
       haStyleDialog,
       css`
         ha-dialog {
-          --mdc-dialog-min-width: 600px;
+          --mdc-dialog-min-width: min(600px, 95vw);
+          --mdc-dialog-max-width: min(600px, 95vw);
```

Comment:
By "move this one", do you mean change mdc-dialog-max-width in haStyleDialog from `600px` to `min(600px, 95vw)`? 

And leave the min-width change here in this file?

Unfortunately I haven't got the faintest idea what a "css fallback" looks like...

Reply:
it would just be a rule that you overwrite,

css```
width: 600px;
width: min(95vw, 600px);
```

A browser that doesnt support `min()` will just skip that rule, so it will be set to 600px, modern browsers will use it.

Reply:
Makes sense to me.